build.gradle.kts

```diff
@@ -150,6 +150,4 @@ tasks {
       jvmTarget = "17"
     }
   }
-
-  named("test") { dependsOn("testIntegration", "testWithSchemaNomis") }
```

Comment:
this meant that when running the `test` profile in intellij integration and test with schema nomis were always run as well